fabric-mining-level-api-v1/src/main/resources/data/fabric/tags/blocks/mineable/shears.json

```diff
@@ -0,0 +1,7 @@
+{
+  "replace": false,
+  "values": [
+    "#fabric:mineable/shears_fast",
+    "#fabric:mineable/shears_slow"
```

Comment:
Does this work when the fast/slow tags are not defined? (In a plain environment without the testmod)

Reply:
Good shout, fixed